WikiDrips/WikiDoc.swift

```diff
@@ -10,19 +10,37 @@ import Foundation
 
 public class WikiDoc
 {
-    public let title = "Test Title String"
-    public let pubDate = Date.init()
+    public let title: String
+    public let pubDate: Date
     public var imageInitials: String {
         get {
             let nameArray = self.title.components(separatedBy: " ")
-            return "\(nameArray.first?.characters.first!)\(nameArray.last?.characters.first!)"
+            var initials = ""
+            
+            if let firstWord = nameArray.first {
+                if let firstInitial = firstWord.characters.first {
+                    initials = "\(firstInitial)"
+                }
+            }
+            
+            if nameArray.count > 1 {
+                if let lastWord = nameArray.last {
+                    if let lastInitial = lastWord.characters.first {
+                        initials += "\(lastInitial)"
+                    }
+                }
+            }
+            
+            return initials
```

Comment:
You can use optional chaining to collapse some of these `if` pyramids such as:
```
    public var imageInitials: String? {
        get {
            let nameArray = self.title.components(separatedBy: " ")
            var initials: String = ""
            
            if let first = nameArray.first?.characters.first {
                initials.append(first)
            }
            
            if let second = nameArray.last?.characters.first, nameArray.count > 1 {
                initials.append(second)
            }
            
            guard initials.isEmpty == false else {
                return nil
            }
            
            return initials
        }
    }
```

I would also change the type to a `String?` so that if we can't come up with initial we probably want to show some default image instead of trying to render an empty string.